recipes-utils/csf-parser/csf-parser_3.4.1.bb

```diff
@@ -0,0 +1,30 @@
+DESCRIPTION = "NXP CSF parser tool"
+HOMEPAGE = "https://www.nxp.com/webapp/Download?colCode=IMX_CST_TOOL"
+LICENSE = "Proprietary"
+LIC_FILES_CHKSUM = " \
+    file://LICENSE.bsd3;md5=14aba05f9fa6c25527297c8aac95fcf6 \
+    file://LICENSE.hidapi;md5=e0ea014f523f64f0adb13409055ee59e \
+    file://LICENSE.openssl;md5=3441526b1df5cc01d812c7dfc218cea6 \
+"
+SRC_URI = "git://git@github.com/balena-os/cst.git;protocol=ssh;branch=master"
```

Comment:
Will this make all device types using `meta-balena-hab` unbuildable for users with no access to the repo?